daemon/tests/happy_path.rs

```diff
@@ -353,15 +353,8 @@ async fn maker_notices_lack_of_taker() {
 /// For convenience, returns also OrderId of the opened Cfd.
 /// `announcement` is used during Cfd's creation.
 async fn start_from_open_cfd_state(announcement: oracle::Announcement) -> (Maker, Taker, OrderId) {
-    let heartbeat_interval = Duration::from_secs(60);
```

Comment:
thanks for cleaning this - this should have been cleared before, I just didn't notice it inside this function! this might have been the reason why some of the tests were failing - `heartbeat_inverval * 2` was a bug (it should have been just `heartbeat_interval`, we changed the API recently).

As such this commit is unrelated to the PR, but still very much desired.